hasher-matcher-actioner/webapp/src/components/GraphWithNumberWidget.jsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// import uPlot from 'uplot';
+import UplotReact from 'uplot-react';
+import '../../node_modules/uplot/dist/uPlot.min.css';
+
+/**
+ * Reduces the width required for displaying a number. eg. 4,000 -> 4k.
+ * 2,000,000 -> 2M etc.
```

Comment:
Should the m be `M` or `m` 
Personally don't have a strong opinion but let's have the comment match the returned value?

Rabbit Hole warning on stack exchange meta: https://meta.stackexchange.com/questions/87251/use-m-for-million-instead-of-m